addons/fire/functions/fnc_medical_success.sqf

```diff
@@ -0,0 +1,23 @@
+#include "script_component.hpp"
+/*
+ * Author: Brandon (TCVM)
+ * Makes object catch fire. Only call from events.
```

Comment:
A few of these function headers don't seem right? Probably holdover from copy + paste